packages/ses/src/lockdown-shim.js

```diff
@@ -166,36 +164,21 @@ export const repairIntrinsics = (options = {}) => {
     d`lockdown(): non supported option ${q(extraOptionsNames)}`,
   );
 
-  // Asserts for multiple invocation of lockdown().
-  if (firstOptions) {
-    for (const name of keys(firstOptions)) {
-      assert(
-        options[name] === firstOptions[name],
-        d`lockdown(): cannot re-invoke with different option ${q(name)}`,
-      );
-    }
-    return alreadyHardenedIntrinsics;
+  if (priorLockdown !== null) {
+    const error = new TypeError(
+      `Already lockded down (SES_ALREADY_LOCKED_DOWN)`,
+    );
+    // eslint-disable-next-line @endo/no-polymorphic-call
+    assert.note(error, assert.details`Prior call ${priorLockdown}`);
+    throw error;
   }
-
-  firstOptions = {
-    dateTaming, // deprecated
-    errorTaming,
-    mathTaming, // deprecated
-    regExpTaming,
-    localeTaming,
-    consoleTaming,
-    overrideTaming,
-    overrideDebug,
-    stackFiltering,
-    domainTaming,
-    __allowUnsafeMonkeyPatching__,
-  };
+  priorLockdown = new TypeError('Prior lockdown (SES_ALREADY_LOCKED_DOWN)');
```

Comment:
FYI this will keep the scope of this function and of its predecessors in the stack around since engines (well at least v8) lazily generate the stack string. While this probably happens early enough in the program execution, it might be worth "touching" the `priorLockdown.stack` for good measure?

Reply:
Really? I have no direct knowledge, but I would have kept only a stack of PC addresses.

Reply:
Apparently JSC automatically generates the stack string if the scope would otherwise have been collected: https://twitter.com/filpizlo/status/1365140028247007234

Maybe v8 could perform some optimization in the case of strict functions and only keep the function name, and location around for `Error.prepareStackTrace`. If you look at the Twitter thread, they apparently ran into some other perf issues.